apps/src/templates/rubrics/LearningGoal.jsx

```diff
@@ -2,27 +2,39 @@ import React, {useState} from 'react';
 import PropTypes from 'prop-types';
 import i18n from '@cdo/locale';
 import style from './rubrics.module.scss';
-import {learningGoalShape} from './rubricShapes';
+import {learningGoalShape, reportingDataShape} from './rubricShapes';
 import FontAwesome from '@cdo/apps/templates/FontAwesome';
 import {
   BodyThreeText,
   BodyFourText,
 } from '@cdo/apps/componentLibrary/typography';
+import analyticsReporter from '@cdo/apps/lib/util/AnalyticsReporter';
+import {EVENTS} from '@cdo/apps/lib/util/AnalyticsConstants';
 
 export default function LearningGoal({
   learningGoal,
   teacherHasEnabledAi,
   canProvideFeedback,
+  reportingData,
```

Comment:
This is a pattern I'm trying out. We often want context information for these events that we don't need for the actual rendering of the component. Definitely open to feedback on this!

Reply:
Tagging @hannahbergam in case you have any thoughts on this!

Reply:
Oooo I like this pattern. Makes me think we could refactor a few other places to match!